package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.15.0",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "node-sass": "^4.0.0",
```

Comment:
move this to devDependencies
for the future - to install devDependency use npm install --save-dev